airbyte-cdk/python/airbyte_cdk/sources/declarative/parsers/factory.py

```diff
@@ -108,7 +108,7 @@ def get_default_type(parameter_name, parent_class):
         type_hints = get_type_hints(parent_class.__init__)
         interface = type_hints.get(parameter_name)
         origin = get_origin(interface)
-        if origin == Union:
+        if origin:
```

Comment:
need to look at the arg whenever there's an origin to the type to also support List[T]

Reply:
sorry I don't follow, wdym?